infra/modules/azure_github_environment_bootstrap/README.md

```diff
@@ -73,11 +73,12 @@ The default branch name can be changed via the `default_branch_name` property.
 
 ## Extending the module for custom needs
 
-The module provides the basic configuration adhering to DX and Technology
-standards. However, it can be extended according to team needs.
+The module provides the basic configuration adhering to DX and Technology standards. However, it can be extended according to new needs. In fact, the module export all the ids and names of the resources that creates, so it is straightforward to add further resources.
```

Comment:
thoughts: the meaning of "extended" is not clear to the casual reader. what do you mean with "extended" in this context?

Reply:
which kind of customization and capabilities? I'd add an example.

Reply:
I'd link "can be extended" to the next paragraph "Customizing GitHub configuration" then. To avoid readers wondering as they read, a best practice when anticipating content is always to link the reference.